style.css

```diff
@@ -101,9 +101,49 @@ aside {
   max-height: 300px;
   display: inline-block;
 }
-
 .productImage {
+
+}
+.productImage:focus {
+  transition: .75s;
+}
+#left:focus{
+  transition: .75s;
+  transform: scale(0)
+}
+#mid{
+  width:243px;
+  height:243px;
+  background-image: url("pictures/iceCreamLock.jpg");
+  outline-width: 0;
+}
+/* background url idea from here https://codemyviews.com/blog/inner-shadows-in-css-images-text-and-beyond*/
+#mid:focus{
+  opacity: .1;
+  transition: .75s;
+  box-shadow: inset 0px 0px 60px 122px rgba(248, 249, 247, .5);
+}
+#right {
+  outline-width: 0;
 }
+#right:focus {
+  animation: boom 1s ease;
+  animation-iteration-count: 1;
+  visibility: visible;
+
+}
```

Comment:
Would love to see this updated with the fix we identified in class to make sure the selected image stays hidden.